src/commands/workflow/run.ts

```diff
@@ -1,24 +1,31 @@
-import { Flags } from "@oclif/core";
+import { Args, Flags } from "@oclif/core";
 
 import * as ApiV1 from "@/lib/api-v1";
 import BaseCommand from "@/lib/base-command";
-import { commaSeparatedStr, jsonStr } from "@/lib/helpers/flag";
+import { jsonStr } from "@/lib/helpers/flag";
 import { withSpinner } from "@/lib/helpers/request";
 import { indentString } from "@/lib/helpers/string";
 
-export default class WorkflowRun extends BaseCommand {
+export default class WorkflowRun extends BaseCommand<typeof WorkflowRun> {
+  static summary =
+    "Test run a workflow using the latest version from Knock, or a local workflow directory.";
+
   static flags = {
     environment: Flags.string({
       default: "development",
       summary: "The environment in which to run the workflow",
     }),
-    recipients: commaSeparatedStr({
+    recipients: Flags.string({
       required: true,
       aliases: ["recipient"],
       summary:
         "One or more recipient ids for this workflow run, separated by comma.",
+      multiple: true,
+      delimiter: ",",
```

Comment:
@meryldakin Sooo it turns out oclif v2 added a `delimiter` option that would parse the string value into an array, which is the same thing we were doing with `commaSeparatedStr` 😅 (but you have to set `multiple: true`). 

I removed the helper in favor of using the built-in delimiter option, but wanted to give you a heads up!

Reply:
totally missed this!! gotcha thanks!